tests/test_s3.py

```diff
@@ -51,3 +53,82 @@ def test_connection_secure():
             http_client=http_client)
 
         assert_true(minio_client.bucket_exists(S3_CONN_INFO['bucket']))
+    @staticmethod
+    @raises(DataJointError)
+    def test_remove_object_exception():
+        # https://github.com/datajoint/datajoint-python/issues/952
+        
+        # Initialize minioClient with an endpoint and access/secret keys.
+        minio_client = Minio(
+            'minio:9000',
+            access_key= 'jeffjeff',
+            secret_key= 'jeffjeff',
+            secure=False)
+
+        minio_admin = minio.MinioAdmin(target = 'myminio')
```

Comment:
PEP8: Here and throughout, remove spaces around `=` in argument lists.

Reply:
entire test now passes PEP8 through flake8 except `E501 line too long` as of commit 08a3fd8